### DISCUSSION TITLE
Welcome to v2rayA Discussions!

### DISCUSSION THREAD

**Discussion by mzz2017** [v2rayA/v2rayA#851] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by falahatiali** [comment]
I insalled and add my server but when I want to connect I get this error: 

failed to connect: failed to connect: geoip.dat or geosite.dat file does not exists

**Comment by mrquokka** [comment]
Hi, cannot install it https://apt.v2raya.org/key/public-key.asc expired at 09.2025

**Comment by wodty** [comment]
2026/02/14 08:00:04.695 [W] [asm_amd64.s:1693] ObservatoryProducer: did not connect: context deadline exceeded
一直提示这个是啥问题

**Comment by AleksanderAnapa** [comment]
Hi , why isn't the fastest server selected based on ping when my subscription is active? How can I manually add this feature to v2raya in openwrt to select the fastest server after renewing my subscription? The first server in the list is always selected, and it is very slow.
